app/widgets/MultiSelect.js

```diff
@@ -10,6 +11,18 @@ import {
   CheckBox,
 } from 'native-base';
 
+/**
+ * getURL will replace an SVG image with a JPG image because
+ * react-native can't handle SVGs, but web prefers them.
+ * @param {String} url 
+ */
+const getURL = (url) => {
+  if (url.endsWith('.svg')) {
+    return url.replace('.svg', '.jpg');
+  }
+  return url;
+};
```

Comment:
It seems like you use this a few times. I would put it in a helper function and import it.

Reply:
done!